File: app/src/main/java/com/bogdanorzea/popularmovies/model/response/ReviewsResponse.java
package com.bogdanorzea.popularmovies.model.response;

import java.util.List;

import com.bogdanorzea.popularmovies.model.object.Review;
import com.squareup.moshi.Json;

public class ReviewsResponse {

    @Json(name = "id")
    public int id;
    @Json(name = "page")
    public int page;
    @Json(name = "results")
    public List<Review> results = null;
    @Json(name = "total_pages")
    public int totalPages;
    @Json(name = "total_results")
    public int totalResults;

}

File: app/src/main/java/com/bogdanorzea/popularmovies/model/response/ReviewsResponse.kt
package com.bogdanorzea.popularmovies.model.response

import com.bogdanorzea.popularmovies.model.`object`.Review
import com.squareup.moshi.Json

class ReviewsResponse(@Json(name = "id") var id: Int = 0,
                      @Json(name = "page") var page: Int = 0,
                      @Json(name = "results") var results: List<Review>? = null,
                      @Json(name = "total_pages") var totalPages: Int = 0,
                      @Json(name = "total_results") var totalResults: Int = 0)
